server/manager/src/main/java/org/apache/accumulo/manager/compaction/coordinator/CompactionCoordinator.java

```diff
@@ -294,10 +294,23 @@ protected void startInternalStateCleaner(ScheduledThreadPoolExecutor schedExecut
     ThreadPools.watchNonCriticalScheduledTask(future);
   }
 
+  protected void startConfigMonitor(ScheduledThreadPoolExecutor schedExecutor) {
+    ScheduledFuture<?> future =
+        schedExecutor.scheduleWithFixedDelay(this::cleanUpInternalState, 0, 1, TimeUnit.MINUTES);
```

Comment:
Should this call `checkForConfigChanges` instead of ` cleanUpInternalState`?

Reply:
ha, yeah it should good catch.  I copied and pasted and changed a few things, but missed that.

Reply:
Fixed that and noticed another bug where it was not remembering the new max size to use w/ new queues that are created after the size was adjusted.   Fixed both in 5471fc7